module/submissions/helper.js

```diff
@@ -1483,4 +1483,326 @@ module.exports = class SubmissionsHelper {
    }
 
 
+    /**
+    * Get criteria questions 
+    * @method
+    * @name getCriteriaQuestions
+    * @param {String} submissionId - submissionId.
+    * @returns {Object} - Criteria questions and answers 
+    */
+
+    static getCriteriaQuestions(submissionId) {
```

Comment:
@deepakr123 - Check for if submissionId is not passed condition.

Reply:
resolved